docs/resources/bms_instance.md

```diff
@@ -0,0 +1,217 @@
+    Changing this creates a new instance.
+
+* `system_disk_size` - (Optional, int, ForceNew) Specifies the system disk size in GB, The value range is 1 to 1024.
+    Shrinking the disk is not supported.
+
+* `data_disks` - (Optional, String, ForceNew) Specifies an array of one or more data disks to attach to the instance.
+    The data_disks object structure is documented below.
+    Changing this creates a new instance.
+
+* `tags` - (Optional, Map) Specifies the key/value pairs to associate with the instance.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique id in UUID format of enterprise project .
+    Changing this creates a new instance.
+
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the instance.
+    Valid values are *prePaid* and *postPaid*, defaults to *prePaid*.
+    Changing this creates a new instance.
+
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the instance.
+    Valid values are *month* and *year*. This parameter is mandatory if `charging_mode` is set to *prePaid*.
+    Changing this creates a new instance.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the instance.
+    If `period_unit` is set to *month*, the value ranges from 1 to 9.
+    If `period_unit` is set to *year*, the value is 1.
+    This parameter is mandatory if `charging_mode` is set to *prePaid*. Changing this creates a new resource.
+
+* `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled.
+    Valid values are "true" and "false", defaults to *false* Changing this creates a new resource.
+
+* `agency_name` - (Optional, String, ForceNew) Specifies the IAM agency name which is created on IAM to provide
+    temporary credentials for BMS to access cloud services. Changing this creates a new instance.
+
+The `nics` block supports:
+
+* `subnet_id` - (Required, String, ForceNew) Specifies the ID of subnet to attach to the instance.
+  Changing this creates a new instance.
+
+* `ip_address` - (Optional, String, ForceNew) Specifies a fixed IPv4 address to be used on this network.
+  Changing this creates a new instance.
+
+
+The `data_disks` block supports:
+
+* `type` - (Required, String, ForceNew) Specifies the BMS data disk type, which must be one of available disk types,
+  contains of *SSD*, *GPSSD* and *SAS*. Changing this creates a new instance.
+
+* `size` - (Required, Int, ForceNew) Specifies the data disk size, in GB. The value ranges form 10 to 32768.
+  Changing this creates a new instance.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - A resource ID in UUID format.
+* `host_id` - The host ID of the instance.
+* `status` - The status of the instance.
+* `description` - The description of the instance.
+* `image_name` - The image_name of the instance.
+* `public_ip` - The EIP address that is associted to the instance.
+
```

Comment:
`nics/mac_address` and `nics/port_id` are missing.

Reply:
fixed